src/index.js

```diff
@@ -91,42 +110,42 @@ export default class ImageTool {
    * @param {object} api - Editor.js API
    */
   constructor({ data, config, api }) {
+    /**
+     * @param {string} src - url on preview image while uploading file
+     */
+    this.setPreview = (src) => {
```

Comment:
I wanted to use https://babeljs.io/docs/en/babel-plugin-proposal-class-properties for this method, but I didn't know if I could add new plugins to your existing presets.